Lib/test/libregrtest/save_env.py

```diff
@@ -285,6 +292,8 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                 restore(original)
                 if not self.quiet and not self.pgo:
                     print_warning(f"{name} was modified by {self.testname}")
-                    print(f"  Before: {original}\n  After:  {current} ",
+                    r1 = self._repr(original)
+                    r2 = self._repr(current)
```

Comment:
Why do you need this change? Could you maybe put in a separated PR if it's not directly related to PEP 578?

Reply:
It was directly related to figuring out which thread was leaking. I don't need it anymore, so either I can revert it or leave it - I'm not filing a new bug for it.

Reply:
To be more explicit, `WeakSet.__repr__` does not show the contents, so when a dangling thread remains this didn't show anything useful.

Reply:
Reverted this and filed bpo-36949